cmd/sync.go

```diff
@@ -55,4 +58,8 @@ func init() {
 		0, "aritificial delay in seconds that is injected between insert operations \n"+
 			"for related entities (usually for cassandra deployments).\n"+
 			"See 'db_update_propagation' in kong.conf.")
+	syncCmd.Flags().IntVar(&syncCmdRetries, "retries",
+		1, "Number of times decK will retry an insert or update operation")
```

Comment:
I'd advise even against making retries configurable in the beginning. I do see value here but I'd like to see users asking for it before we add it. Having an internally hardcoded value of `5` retries should do it for most users.

Reply:
Makes sense, I've experimented with some values for the updated implementation with exponential backoff and came up with 3 retries after around 1 second, 3 seconds, and 9 seconds (randomized to avoid the thundering herd). Let me know what you think.